### PR TITLE
vendor: correctly update golang client

### PR DESCRIPTION
**Pull request by si74**
oops! had a bit of a mistake in vendoring this last time. this should do the trick now.

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -741,17 +741,29 @@
 			"revision": "d77da356e56a7428ad25149ca77381849a6a5232",
 			"revisionTime": "2016-06-15T09:26:46Z"
 		},
+		{
+			"checksumSHA1": "4VppKBbzCSmoFfQxGNUm0TYiFCA=",
+			"path": "github.com/prometheus/client_golang/api",
```
Comment (brian-brazil) [inline vendor/vendor.json]: Why are you pulling these in? We don't use them.

Comment (si74) [reply vendor/vendor.json]: I'm working a particular open task (adding query to promtool) and was hoping to use these for them.

Comment (si74) [reply vendor/vendor.json]: This one! https://github.com/prometheus/prometheus/issues/1605

**Comment by brian-brazil** [comment]
Usually we only add new dependencies in the PR that needs them.

**Comment by si74** [comment]
oops! thanks for the heads up - will do so next time

**Pull request merged**